asn1/README.md

```diff
@@ -0,0 +1,7 @@
+# Important Notice
+
+This is a Fork of a Go Package. The original source can be found on
```

Comment:
```suggestion
This is a fork of the `encoding/asn1` Go package. The original source can be found on
```

Reply:
Yeah, was being lazy so I could just copy the file.